ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -33,6 +33,29 @@ images:
     RUN chmod +x /usr/local/bin/oc
   from: src
   to: openstack-operator-ci-build-deploy
+- dockerfile_literal: |
+    FROM centos/centos:stream9
+    USER root
+    RUN dnf install -y gcc git jq make python3 python3-pip python-setuptools python-requests
+    RUN git clone https://git.openstack.org/openstack/tripleo-repos && \
+        pushd tripleo-repos && \
+        python3 setup.py install && \
+        popd && \
+        /usr/local/bin/tripleo-repos current-tripleo-dev
+
+    RUN git clone https://git.openstack.org/openstack/openstack-tempest-skiplist && \
+        pushd openstack-tempest-skiplist && \
+        python3 -m pip install . && \
+        popd
+
+    RUN dnf update -y && \
+        dnf install -y python3-tempestconf openstack-tempest-all && \
+        dnf clean all && \
+        rm -rf /var/cache/dnf/*
+    RUN curl -s -L https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | tar xvzf - -C /usr/local/bin oc
+    RUN chmod +x /usr/local/bin/oc
+  from: src
```

Comment:
the issue with "from: src"is that it will replace the "FROM centos/centos:stream9" in your Dockerfile. Something that you probably don't want to do.
You can just omit this "from: src" line, since your tempest step don't depends on the operator code.